bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClientWrapper.java

```diff
@@ -196,23 +197,34 @@ public Table apply(com.google.bigtable.admin.v2.Table tableProto) {
   /** {@inheritDoc} */
   @Override
   public void dropRowRange(String tableId, String rowKeyPrefix) {
-    DropRowRangeRequest requestProto = DropRowRangeRequest.newBuilder()
+    DropRowRangeRequest.Builder dropRequestProtoBuiler = DropRowRangeRequest.newBuilder()
```

Comment:
is the dropRequestProtoBuiler intentional?

Reply:
I meant why not dropRequestProtoBuilder?

Reply:
Thanks!!